### PR TITLE
Feature/new scheduled booking

### PR DESCRIPTION
**Pull request by evbambly**
image
image

New Bookings are now shown as new.
If Only New Bookings button is selected, 
new bookings/cancellations are shown without a tag.

### CODE REVIEW

File: packages/coordinator/src/components/Chip/Chip.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  label: string;
+  /* Standard template chips **/
+  chipType?: StandardChip;
+  color?: string;
+  backgroundColor?: string;
+  fontSize?: string;
+  /* Applying width disables default padding on label span **/
+  width?: string;
+  height?: string;
+};
```
Comment (yaron1m) [inline packages/coordinator/src/components/Chip/Chip.tsx:82]: 2 comments:
1. If we have `chipType` why do we also need `label`?
2. I don't like doing הכנה למזגן. If currently no one needs custom height and width, I don't think we should support such props. When we need them, we can add them

File: packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsScreen.tsx [hunk 2]
```diff
@@ -32,6 +32,7 @@ interface ManageAppointmentsScreenProps {
   onDeleteAppointment: (appointmentId: string) => Promise<void>;
   onRemoveDonor: (appointmentId: string) => Promise<void>;
   isLoading: boolean;
+  hasOnlyNewChanges: boolean;
```
Comment (yaron1m) [inline packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsScreen.tsx:35]: showOnlyWithRecentChanges

File: packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsScreenContainer.tsx [hunk 3]
```diff
@@ -20,6 +20,7 @@ export default function ManageAppointmentsScreenContainer() {
     getDefaultState()
   );
   const [isLoading, setIsLoading] = useState(false);
+  const [showOnlyNewChanges, setShowOnlyNewChanges] = useState(false);
```
Comment (yaron1m) [inline packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsScreenContainer.tsx:23]: Please go over the PR and be consistent with the naming. 
Now we have `showOnlyNew`, `ChangeType`, `BookingChange` and they all refer to the same thing

File: packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsTableConfig.tsx [hunk 4]
```diff
@@ -29,11 +31,35 @@ export const GetExpandedColumns = (
...
+    cellRenderer: ({ booked, bookingTimeMillis, recentChangeType }) => {
+      let bookingDate = "אין רישום";
+      if (booked && bookingTimeMillis) {
+        bookingDate =
+          "נקבע בתאריך " + DateUtils.ToDateString(bookingTimeMillis);
       }
-      return "נקבע בתאריך " + DateUtils.ToDateString(bookingTimeMillis);
+      const chipLabel =
+        recentChangeType === BookingChange.CANCELLED ? "בוטל" : "נקבע";
+      return (
```
Comment (yaron1m) [inline packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsTableConfig.tsx:42]: Should this just be a new column?

This way we could possibly even sort by it.

WDYT?

File: packages/coordinator/src/components/Chip/Chip.tsx (hunk 1 above)
Comment (evbambly) [reply packages/coordinator/src/components/Chip/Chip.tsx:82]: ChipType's name is now ColorScheme for clarity.

File: packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsScreenContainer.tsx (hunk 3 above)
Comment (evbambly) [reply packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsScreenContainer.tsx:23]: showOnlyNew converted to showOnlyRecentChanges,
in consistency with RecentChangeType.

BookingChange refers to both recent and non-recent changes

File: packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsTableConfig.tsx (hunk 4 above)
Comment (evbambly) [reply packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsTableConfig.tsx:47]: I agree, it might be more convenient

**Review by yaron1m — APPROVED** [review]

**Pull request merged**